integration-tests/actions/actions.go

```diff
@@ -258,20 +252,7 @@ func TeardownSuite(
 		}
 	}
 
-	switch strings.ToUpper(keepEnvs) {
-	case "ALWAYS":
-		return nil
-	case "ONFAIL":
-		if !t.Failed() {
-			return env.Shutdown()
-		}
-	case "NEVER":
-		return env.Shutdown()
-	default:
-		log.Warn().Str("Invalid Keep Value", keepEnvs).
-			Msg("Invalid 'keep_environments' value, see the KEEP_ENVIRONMENTS env var")
-	}
-	return nil
+	return env.Shutdown()
```

Comment:
do we not need `keepEnvs` logic anymore?

Reply:
The keepEnvs logic now exists in chainlink-env https://github.com/smartcontractkit/chainlink-env/pull/215/files#diff-6c86ee279425279a568c6320829a514721c13cb2227ff0014f5fc75fd28ce5b1R543-R580